glean-core/src/core_metrics.rs

```diff
@@ -26,6 +28,8 @@ pub struct ClientInfoMetrics {
     pub channel: Option<String>,
     /// The Android specific SDK version of the software running on this hardware device (e.g. "23").
     pub android_sdk_version: Option<String>,
+    /// The Windows specific OS build version (e.g. "19043")
```

Comment:
(since it isn't a string)
```suggestion
    /// The Windows specific OS build version (e.g. 19043)
```